docs/src/security.md

```diff
@@ -26,6 +26,25 @@ a linter that scans the abstract syntactic tree of the source against a set of r
 the discovery of well-known vulnerabilities, threats, and weaknesses hidden in
 the code such as hard-coded credentials, integer overflows and SQL injections - to name a few.
 
+Another step during the CI/CD pipeline is running [govulncheck](https://pkg.go.dev/golang.org/x/vuln/cmd/govulncheck)
+which reports known vulnerabilities that affect Go code, or even the compiler, meaning that if
+for some reason, the operator is built with a version of the Go compiler that contains
+a known bug this will be spotted during the process.
+
+GitHub also provides another tool [CodeQL](https://codeql.github.com/) that has been enabled
+and included in our CI/CD pipeline, thus, will block any pull request on which it could find
+any security issue. The configuration of CodeQL was set to only review Go code, so any other
+language in the repository like, python or bash, is not covered by this test.
+
+The [Snyk](https://snyk.io/) has an Open Source Projects in which CloudNativePG was accepted and
+we run the Snyk code scan every night in a scheduled job and produce weekly reports with any
+new finding on code security and also for licensing issues that could come up.
+
+One of the last things to note is that we have the "Report a vulnerability" option enabled
+on the Security section inside the repository that allows users to safely report any security
```

Comment:
I don't get this. We do have documentation explaining how to report vulnerabilities.
Do you mean, that section? If so I'll link here.
Been looking at the repo on GH and I don't see any vulnerability reporting tooling.